commands/lily.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/filecoin-project/lily/lens/lily"
 )
 
-func GetAPI(ctx context.Context, addrStr string, token string) (lily.LilyAPI, jsonrpc.ClientCloser, error) {
-	addrStr = strings.TrimSpace(addrStr)
+func GetAPI(ctx context.Context) (lily.LilyAPI, jsonrpc.ClientCloser, error) {
```

Comment:
looks at flags rather than accept them as arguments, decrease the opportunity for developer error.